docs/recipes.md

```diff
@@ -560,7 +552,7 @@ import functools
 @functools.cache
 def compute_predictions(problem_parameters):
    # replace with your own function/parameters
-	...
+ ...
```

Comment:
@mscolnick was this change made by the pre-commit? The ellipses should be indented.